css/main.css

```diff
@@ -2,321 +2,517 @@ body {
 	background-color: #f2f2f2;
 	font-family: 'PT Sans', sans-serif;
 	margin: 0;
-	padding: 0 40px;
+	padding: 0;
 }
 
-.container {
-	max-width: 1200px;
-	box-sizing: border-box;
+.wrapper {
+	width: 1200px;
+	margin: 0 auto;
 	background-color: #ffffff;
-	margin: 0;
-	padding: 0;
 }
 
-header {
+.header {
 	background-image: url('../img/header.png');
 	background-position: bottom;
 	background-repeat: no-repeat;
-
 	height: 360px;
 	max-width: 1200px;
 }
 
-ul {
-	list-style: none;
-	margin: 0;
-	padding: 0;
+.heading-text {
+	color: #ffffff;
 	text-transform: uppercase;
+	font-size: 5rem;
+	font-weight: 800;
+	text-align: center;
+	position: relative;
+	top: 3rem;
 }
 
-nav {
-	background-color: #ffffff;
-	margin: 0;
-	padding: 0;
-}
 
-nav a {
-	text-decoration: none;
-	margin: 0;
-	color: #000000;
+.logo {
+	display: block;
+	position: relative;
+	left: 50%;
+	transform: translateX(-94px)
```

Comment:
don't use pixels, transform allows to use calc(50%)

something like: 
transform: translateX(calc(50%));